gulpfile.ts

```diff
@@ -0,0 +1,25 @@
+/// <reference path="typings/tsd.d.ts"/>
+/// <reference path="./typings/shelljs/shelljs.d.ts"/>
+
+import gulp = require('gulp');
+import ts = require('gulp-typescript');
+import shell = require("shelljs");
+
+var isWin = /^win/.test(process.platform);
+
+// OMG! This is so dirty I need to take a shower!
+process.env["NODE_PATH"] = "./build/" + (isWin ? ";" : ":") + process.env["NODE_PATH"];
```

Comment:
OMG! Thank you very much, uncle Never! It was 3 am and I was sleepy as hell and I was simply running out of googling power. :)